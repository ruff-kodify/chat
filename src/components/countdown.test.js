import React from 'react';
import Countdown from './countdown';

test('it should render properly if value is set', () => {
  const wrapper = shallow(
    <Countdown
      value={ 5 }
    />
  );
  expect(wrapper).toMatchSnapshot();
});

test('it should render properly if value is not set', () => {
  const wrapper = shallow(
    <Countdown />
  );
  expect(wrapper).toMatchSnapshot();
});
