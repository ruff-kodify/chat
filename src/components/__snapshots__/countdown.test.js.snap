// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it should render properly if value is not set 1`] = `
<div
  className="countdown"
/>
`;

exports[`it should render properly if value is set 1`] = `
<div
  className="countdown"
>
  5
</div>
`;
